aries-site/src/pages/design/index.js

```diff
@@ -27,7 +27,58 @@ const Design = () => {
       {page.pages.length ? (
         <NavPage items={page.pages} topic={page.name.toLowerCase()} />
       ) : (
-        <ComingSoon />
+        <>
+          <ContentSection>
+            <Subsection name="Preferred">
```

Comment:
I think a more descriptive title would be helpful -- it left me thinking "Preferred what?"